UPDATE draft site conf 2018 no publised

cristofer-dev opened the pull request.

cristofer-dev committed:
ADD draft landing conference 2018

 Changes to be committed:
	modified:   evento2017.html
	new file:   evento2018.html

cristofer-dev committed:
MODIFY request change by Dmaster68

 Changes to be committed:
	modified:   evento2018.html

DMasterIO committed:
Merge pull request #35 from iv-devs/FEATURE/add-event-2018

ADD draft landing conference 2018

DMasterIO commented on evento2018.html:
```diff
@@ -0,0 +1,165 @@
+
+  <div id="numeros" class="row ">
+    <div class="col-md-12  text-center ">
+      <h1>IVDevs en números</h1>
```
le agregaría el espacio onda IV Devs, nunca me ha gustado como queda junto

DMasterIO commented on evento2018.html:
```diff
@@ -0,0 +1,165 @@
+      <span>La comunidad de NodeJS y JS en Lationamérica</span>
+    </div>
+    <div class="col-md-4 col-sm-4 col-xs-12 text-center img">
+      <img src="/assets/img/partners/logo_microsoft.png" alt="">
```
antes de pasarla a master confirmaría este tema, no es menor poner a Microsoft como partner y que no sepan.. prefiero quitarlos

cristofer-dev replied:
Yo lo sacaria!

DMasterIO replied:
yo también

cristofer-dev replied:
DONE

DMasterIO commented on evento2018.html:
```diff
@@ -0,0 +1,165 @@
+      </p>
+    </div>
+    <div class="row">
+
```
son: (próximamente || Por Definir || En proceso de...)
La idea es dejar enclaro que viene algo interesante, pero que aún analizamos por eso no quitaría la sección pero si dejaría un "banner" o texto que deje en claro el proceso

cristofer-dev replied:
Ok!

DMasterIO replied:
si me haces una imagen bonita que poner ahí, yo mismo saco los cambios

cristofer-dev commented:
@hpaez @CepheiA @DMaster69 @rodpoblete quien apaña con los cambios ?

cristofer-dev committed:
UPDATE content sponsorship proposal

 Changes to be committed:
	new file:   assets/img/LogoDevConf2018.svg
	modified:   evento2018.html

cristofer-dev committed:
DELETE Logo MS in partners

 Changes to be committed:
	modified:   evento2018.html

DMasterIO committed:
Merge pull request #39 from iv-devs/FEATURE/add-event-2018

Feature/add event 2018
